backend/app/store/service/service.go

```diff
@@ -104,6 +107,10 @@ func (s *DataStore) Create(comment store.Comment) (commentID string, err error)
 	commentID, err = s.Engine.Create(comment)
 	s.submitImages(comment)
 
+	if e := s.indexComment(comment); e != nil {
+		log.Printf("[ERROR] failed to index comment, %v", e)
```

Comment:
I don't think you meant to error here and terminate the flow. This failure is not critical, should be WARN